data/meson.build

```diff
@@ -2,7 +2,10 @@ subdir('builder')
 subdir('pki')
 subdir('remotes.d')
 subdir('bash-completion')
+
+if get_option('fish_completion')
```

Comment:
I think this should group with bash completion. You don't want any completions for an embedded target probably.

Reply:
maybe if `get_option('completion')` instead? Or add a `bash_completion` one too?

Reply:
I say the former

Reply:
Although, there seems to be some precedent for `-Dbash_completion=false` looking at the meta-openembedded tree.

Reply:
I've added the latter to match the others.